web-app/js/portal/details/NCWMSColourScalePanel.js

```diff
@@ -39,6 +39,9 @@ Portal.details.NCWMSColourScalePanel = Ext.extend(Ext.Panel, {
         });
 
         this.items = [
+            new Ext.Spacer({
+                height: 10
+            }),
```

Comment:
I went out of my way to specifically place it here. if there will be some HTML from the NCWMSColourScalePanel then i want the space. @dnahodil